__init__.py

```diff
@@ -25,12 +24,11 @@ def runtime_requirements(self):
                                    no_network_fallback=True,
                                    no_gui_fallback=True)
 
-    def initialize(self):
-        """ Perform any final setup needed for the skill here.
-        This function is invoked after the skill is fully constructed and
-        registered with the system. Intents will be registered and Skill
-        settings will be available."""
-        my_setting = self.settings.get('my_setting')
+    @property
+    def get_my_setting(self):
+        """Dynamically get the my_setting from the skill settings file.
+        If it doesn't exist, return the default value."""
```

Comment:
if you want to add a note as for why this should be done, mention that this will reflect live changes to settings.json files  (local or from backend)